### ISSUE TITLE
Conch itself uses deprecated keys APIs

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3153]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)| @jml reported|
|-|-|
|Trac ID|trac#3153|
|Type|defect|
|Created|2008-04-03 07:00:49Z|

While upgrading Twisted, I noticed that Conch itself is using APIs that are deprecated in Conch 0.9 (don't know when that release was).

It should be using the APIs that it recommends.

Attachments:

* [conch-keys.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b9b/b9ba2e4dc4732c9114cb035fea22fb41dbdbd067/e804da6d3ffcfa2f962d6576172ac0662946e385.diff) (1357 bytes) - added by jml on 2008-04-03 07:01:40Z - Updates some of the uses of deprecated APIs (no tests)

<details><summary>Searchable metadata</summary>

```
trac-id__3153 3153
type__defect defect
reporter__jml jml
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1207206049000000 1207206049000000
changetime__1239923750000000 1239923750000000
version__None None
owner__ 
cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_1"></a>|@jml commented|
|-|-|

This patch sponsored by Canonical Ltd.

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_2"></a>|@jml set status to `closed`|
|-|-|

Fixed independently by z3p in r26534 as part of the fix for [#2682](https://github.com/twisted/twisted/issues/2682).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
